packages/twenty-front/src/modules/object-record/record-index/states/recordIndexRecordGroupHideComponentState.ts

```diff
@@ -0,0 +1,9 @@
+import { createComponentStateV2 } from '@/ui/utilities/state/component-state/utils/createComponentStateV2';
+import { ViewComponentInstanceContext } from '@/views/states/contexts/ViewComponentInstanceContext';
+
+export const recordIndexRecordGroupHideComponentState =
+  createComponentStateV2<boolean>({
+    key: 'recordIndexRecordGroupHideComponentState',
+    defaultValue: false,
+    componentInstanceContext: ViewComponentInstanceContext,
```

Comment:
I think we should use RecordIndexComponentContext here but not sure we have migrated yet